bayes_people_tracker/src/bayes_people_tracker/people_tracker.cpp

```diff
@@ -627,7 +627,7 @@ void PeopleTracker::connectCallback(ros::NodeHandle &n) {
         for(it = subscribers.begin(); it != subscribers.end(); ++it)
             subscribers[it->first] = n.subscribe<geometry_msgs::PoseArray>(it->first.second.c_str(), 1000, boost::bind(&PeopleTracker::detectorCallback, this, _1, it->first.first));
         for(it = subscribers_people.begin(); it != subscribers_people.end(); ++it)
-            subscribers_people[it->first] = n.subscribe<people_msgs::People>(it->first.second.c_str(), 1000, boost::bind(&PeopleTracker::detectorCallback_people, this, _1, it->first.first));
+            subscribers_people[it->first] = n.subscribe<bayes_people_tracker::PeopleStamped>(it->first.second.c_str(), 1000, boost::bind(&PeopleTracker::detectorCallback_people, this, _1, it->first.first));
```

Comment:
Sorry, but this change breaks the system for many other projects, as it changes the type of a subscriber. That means that all our other projects will not work with this anymore, as they are publishing `people_msgs::People`. I must insist you create a separate subscriber for this, not change an existing one in an non-compatible way.

Reply:
Fixed now with separate subscriber for peoplestamped msg type.